src/main/java/Main.java

```diff
@@ -1,9 +1,21 @@
 /**
  * Created by iyasuwatts on 10/17/17.
  */
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args ){
 
+        Scanner sc = new Scanner(System.in);
+        String usersName;
+
+        System.out.println("Hello! Please enter your name: ");
+        usersName = sc.nextLine();
```

Comment:
It would read a bit cleaner if we declared and assigned our variable simultaneously here.
(This is not always the case)
`String userName = sc.nextLine();`